src/actions.cpp

```diff
@@ -656,19 +642,23 @@ void evm_contract::call_(intx::uint256 s, const bytes& to, intx::uint256 value,
         txn.to = to_evmc_address(bv_to);
     }
 
-    push_tx(txn, current_version);
+    runtime_config rc;
+    rc.allow_special_signature = true;
+    rc.abort_on_failure = true;
+    rc.enforce_chain_id = false;
+    rc.allow_non_self_miner = false;
+
+    transaction tx{txn};
+    dispatch_tx(rc, tx);
```

Comment:
`dispatch_tx(rc, transaction{std::move(txn)});`.

Reply:
Also, I think the `runtime_config` should be defined in the `call` and `admincall` actions separately (even though they are the same right now) and then passed through the `call_` helper function.